api/fixtures/example.go

```diff
@@ -225,6 +224,7 @@ web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
 				},
 			},
 		}
+		resources = awsResources
```

Comment:
Why not directly assign to `resources` as before? This is pretty confusing to read

Reply:
fixed both the review comments, ptal latest commit